examples/spring-boot/marketplace-app/src/main/java/com/example/vendasta/ShoeStore/utils/Constants.java

```diff
@@ -0,0 +1,12 @@
+package com.example.vendasta.ShoeStore.utils;
+
+
+public class Constants {
+    public static final String APPID = "< APP-ID >";
+    public static final String PARTNERID = "< PARTNER-ID >";
+    public static final String SERVICE_ACCOUNT_JSON_PATH = "< SERVICE-ACCOUNT-JSON-PATH >";
```

Comment:
These can likely be moved to application.yml so that there is only one place to enter the environment variables. It [looks like](https://www.baeldung.com/spring-boot-yaml-vs-properties#3-configurationproperties-annotation) there is a helper to autoload the values into a struct if you want.

We could nest them under the apigateway category. Ex
```
apigateway:
  environment: demo
  app-id: MP-PC8QFF45MSF8QKTZK2RD8ZDBHZJ72GXM
  partner-id: 9YW9
  service-account-json-path: /Users/rrance/.config/api-gateway/demo-9YW9.json
```